redwood-lazylayout-uiview/src/commonMain/kotlin/app/cash/redwood/lazylayout/uiview/UIViewLazyList.kt

```diff
@@ -315,6 +315,7 @@ internal class LazyListContainerCell(
     super.prepareForReuse()
     binding?.unbind()
     binding = null
+    content = null
```

Comment:
Unfortunately `prepareForReuse()` isn’t called reliably by UITableView. I would prefer for it to be called when it puts a cell in its pool, but unfortunately it don’t think it calls it until it removes a cell from its pool.

Reply:
Pushed an alternative approach